ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.qml

```diff
@@ -44,11 +44,37 @@ Rectangle {
         id: mapView
         anchors.fill: parent
         
+        Credential {
+            id: credentials
+            username: "viewer01"
+            password: "I68VGU^nMurF"
+        }
+
         Map {
+
```

Comment:
If no properties follow the component declaration then it doesn't need a new line before the next component is declared.
```suggestion
```